crates/backend/src/ast.rs

```diff
@@ -1072,3 +1105,24 @@ fn replace_self(name: &Ident, item: &mut syn::ImplItem) {
 
     syn::visit_mut::VisitMut::visit_impl_item_mut(&mut Walk(name), item);
 }
+/// Extract the documentation comments from a Vec of attributes
+fn extract_doc_comments(attrs: &Vec<syn::Attribute>) -> Vec<String> {
```

Comment:
Is there a reason this needs to take `&Vec<syn::Attribute>` rather than `&[syn::Attribute]`? If not, then we should switch to the latter.

Reply:
There is not reason, just updated it